R/estimateCellTypeFromProjectionPerCluster.R

```diff
@@ -13,24 +14,27 @@
 #' @export
 #'
 
-estimateCellTypeFromProjectionPerCluster <- function(rca.obj, homogeneity=NULL) {
+estimateCellTypeFromProjectionPerCluster <- function(rca.obj, homogeneity=NULL,vec=F) {
```

Comment:
Vectorization is optional for `estimateCellTypeFromProjectionPerCluster`. Set `vec=FALSE` to default to original implementation.